dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/RemoteAPITokenFrom.java

```diff
@@ -0,0 +1,96 @@
+package com.dotcms.rest.api.v1.authentication;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.map;
+
+@JsonDeserialize(builder = RemoteAPITokenFrom.Builder.class)
+public class RemoteAPITokenFrom {
```

Comment:
Rename this to `form` instead of `from`

Reply:
done https://github.com/dotCMS/core/pull/19643/commits/0c8f8477e3a1c3d9654944171e3904c4f27b06e0#diff-22a926c9d121c69fa55d7a1e39a2619e9371629fd92f245bf0f416b74690bdbbR11